File: C++17/백준/Bronze/9085. 더하기/더하기.cc
#include <iostream>
using namespace std;

int main () {
    int t;
    cin >> t;
    
    for (int i = 0; i < t; i++) {
        int n;
        cin >> n;
        
        int total_sum = 0;
        
        for (int j = 0; j < n; j++) {
            int num;
            cin >> num;
            total_sum += num;
        }
        cout << total_sum << endl;
    }
    return 0;
}

File: Java/백준/Bronze/9085. 더하기/더하기.java
import java.util.Scanner;

public class Main {
    public static void main (String[] args) {
        Scanner sc = new Scanner (System.in);
        
        int t = sc.nextInt();
        
        for (int i = 0; i < t; i++) {
            int n = sc.nextInt();
            
            int total_sum = 0;
            
            for (int j = 0; j < n; j++) {
                int num = sc.nextInt();
                total_sum += num;
            }
            System.out.println(total_sum);
        }
        sc.close();
    }
}